### ISSUE TITLE
View More - Location 

### ISSUE THREAD

**Issue by bmrgould** [gctools-outilsgc/gccollab-mobile#186]
For pages with lists of content, and view more buttons:

Currently the div for content is it's own thing, and new content is appended to it. Then the view-more button is a separate div after the content div. 

This is an issue because a screen reader will reach the more button, and if the user activates it, the content will be loaded behind their current location. Meaning they cannot properly navigate through the content.

This can be fixed by having the more button within the content div, and removing the current one when adding more content. Then appending another one at end of the new content.
